pkg/skaffold/runner/deployer.go

```diff
@@ -86,13 +90,28 @@ func GetDeployer(runCtx *v2.RunContext, provider deploy.ComponentProvider, label
 			deployers = append(deployers, deployer)
 		}
 
+		// TODO: deprecate KustomizeDeploy
```

Comment:
```suggestion
		// TODO(yuwenma)[07/23/21]: deprecate KustomizeDeploy
```

Reply:
ack. This TODO is removed in the next PR which is diff-based from this one. I'll leave the TODO untouched so as there's one less merge-conflict to fix 😄